lib/src/database/indexeddb_box.dart

```diff
@@ -70,21 +70,23 @@ class BoxCollection with ZoneTransactionMixin {
         return;
       });
 
-  Future<void> clear() async {
+  Future<void> clearDb() async {
     final txn = _db.transaction(boxNames.toList(), 'readwrite');
     for (final name in boxNames) {
       unawaited(txn.objectStore(name).clear());
     }
     await txn.completed;
   }
 
-  Future<void> close() async {
+  Future<void> closeDb() async {
     assert(_txnCache == null, 'Database closed while in transaction!');
     return _db.close();
   }
 
-  static Future<void> delete(String path, [dynamic factory]) =>
-      (factory ?? window.indexedDB!).deleteDatabase(path);
+  Future<void> deleteDb(String name, [dynamic factory]) async {
```

Comment:
Is there any context for why those got renamed?

Reply:
cleaned up